go/src/gosetup/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"os"
+	"path"
+	"net/http"
+	"io"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/samalba/dockerclient"
+)
+
+// instance of DockerClient allowing for making calls to the docker daemon
+// remote API
+var DOCKER_CLIENT *dockerclient.DockerClient
```

Comment:
nit: consts and globals should be CamelCase in golang
nit: comment for exported variables should begin with `VariableName ...`